src/browser.ts

```diff
@@ -130,10 +135,21 @@ export class GrpcClient {
   }
 };
 
-lro.ALL_SCOPES = ALL_SCOPES;
 
 export function lro(options) {
     options = Object.assign({scopes: lro.ALL_SCOPES}, options);
     const gaxGrpc = new GrpcClient(options);
     return new OperationsClientBuilder(gaxGrpc);
 }
+
+// Wrapper function to throw exceptions on unsupported streaming calls
+export function createApiCall(  
+  func: Promise<GRPCCall> | GRPCCall,
+  settings: gax.CallSettings,
+  descriptor?: Descriptor
+): GaxCall {
+  if (typeof descriptor != "undefined" && descriptor.constructor.name == "StreamDescriptor"){
```

Comment:
Please use `!==` and `===`. Yes it's JavaScript :)  More info [here](https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons?page=1&tab=votes#tab-top).

Reply:
Done.